content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -307,22 +307,22 @@ That's your Pod's IP address.
 
 Query the Kubelet's API at `http://<IP>:<Port>/pods`.
 
-For this example, it's `http://10.241.1.1:10250/pods`
+In this example, the Pod's URL is `http://10.241.1.1:10250/pods`:
```

Comment:
No. This URI is the kubelet's API endpoint. It is not the URL of a Pod.

Reply:
Done